sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -919,6 +927,34 @@ public Symbol visitObjectLiteral(ObjectLiteral node, ExpressionAnalysisContext c
                 context);
         }
 
+        private final Map<IntervalLiteral.IntervalField, IntervalParser.Precision> INTERVAL_FIELDS =
```

Comment:
```suggestion
        private static final Map<IntervalLiteral.IntervalField, IntervalParser.Precision> INTERVAL_FIELDS =
```

Reply:
Inner classes cannot have static members, but maybe I remove `IntervalLiteral.IntervalField` completely and use the `IntervalParser.Precision` everywhere, then the whole map can go away.